### PR TITLE
K8s configuation

### PR DESCRIPTION
**Pull request by anantnema**

### CODE REVIEW

File: k8s/production/vars.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+prefix: prod
+namespace: 'consumer'
+fqdn: 'rendertron.private.pharmeasy.in'
+tag: '{{ prefix }}-{{ build_number }}'
+context: production
+env_during_build: True
+profile: default
+min_replicas: 12
+max_replicas: 50
+workload_type: compute
```
Comment (johnson-prabu) [inline k8s/production/vars.yml:12]: pls change it to workload_type: general

File: k8s/beta/vars.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+env: production
```
Comment (johnson-prabu) [inline k8s/beta/vars.yml:1]: pls check whether beta is required,  pls change the variables accordingly

File: k8s/production/vars.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+env: production
+project_name: rendertron
+prefix: prod
+namespace: 'consumer'
+fqdn: 'rendertron.private.pharmeasy.in'
+tag: '{{ prefix }}-{{ build_number }}'
+context: production
+env_during_build: True
+profile: default
+min_replicas: 12
```
Comment (johnson-prabu) [inline k8s/production/vars.yml:10]: Kindly review min & max replica count

File: k8s/beta/vars.yml (hunk 2 above)
Comment (anantnema) [reply k8s/beta/vars.yml:1]: required

File: k8s/beta/vars.yml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 env: production
 project_name: rendertron
 namespace: 'beta-consumer'
-fqdn: '{{ prefix }}.pharmeasy.in'
+fqdn: '{{ prefix }}-rendertron.private.pharmeasy.in'
```
Comment (johnson-prabu) [inline k8s/beta/vars.yml:4]: {{ project_name }}

File: k8s/qa/vars.yml [hunk 5]
```diff
@@ -1,7 +1,7 @@
 env: qa
 project_name: rendertron
 namespace: 'qa-consumer'
-fqdn: '{{ prefix }}-{{ project_name }}.qa.pharmeasy.in'
```
Comment (johnson-prabu) [inline k8s/qa/vars.yml]: pls keep "qa.pharmeasy.in"

File: k8s/development/vars.yml [hunk 6]
```diff
@@ -1,7 +1,7 @@
 env: development
 project_name: rendertron
 namespace: 'dev-consumer'
-fqdn: '{{ prefix }}-{{ project_name }}.dev.pharmeasy.in'
```
Comment (johnson-prabu) [inline k8s/development/vars.yml]: pls use "dev.pharmeasy.in"

File: k8s/development/vars.yml [hunk 7]
```diff
@@ -1,7 +1,7 @@
 env: development
```
Comment (johnson-prabu) [inline k8s/development/vars.yml:1]: please add "prefix: dev"

**Review by johnson-prabu — APPROVED** [review]

**Pull request merged**